dbusmock/__main__.py

```diff
@@ -43,6 +45,10 @@ def parse_args():
                         help='automatically implement the org.freedesktop.DBus.ObjectManager interface')
     parser.add_argument('-p', '--parameters',
                         help='JSON dictionary of parameters to pass to the template')
+    parser.add_argument('-x', '--shell', action='store_true',
+                        help='Open a SHELL with the given mock for debugging purposes')
+    parser.add_argument('-e', '--exec', nargs=argparse.REMAINDER,
+                        help='Command to run in the mock environment')
```

Comment:
I find this too redundant. Surely you could just do `-e bash` to get a shell? That's pretty much what the code does anyway.

Reply:
That's fine... It was just a convenience command, I can drop it.